Gen2-Sample/CSharp-TSI-Gen2-Sample/DataPlaneClient/azure-rest-api-specs/specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/examples/QueryAggregateSeriesWithCategoricalVariable.json

```diff
@@ -0,0 +1,238 @@
+{
+  "parameters": {
+    "api-version": "2020-07-31",
+    "environmentFqdn": "10000000-0000-0000-0000-100000000109.env.timeseries.azure.com",
+    "continuationToken": "aXsic2tpcCI6MTAwMCwidGFrZSI6MTAwMH0=",
+    "parameters": {
+      "aggregateSeries": {
+        "searchSpan": {
+          "from": "2019-10-10T23:42:00.000Z",
+          "to": "2019-10-10T23:42:20.000Z"
+        },
+        "timeSeriesId": [
+          "Sensor_58"
+        ],
+        "interval": "PT2S",
+        "inlineVariables": {
+          "Status_String": {
+            "kind": "categorical",
+            "value": {
+              "tsx": "$event.[Status].String"
+            },
+            "categories": [
+              {
+                "label": "Good",
+                "values": [
+                  "Good",
+                  "Very Good",
+                  "Excellent"
+                ]
+              },
+              {
+                "label": "Bad",
+                "values": [
+                  "Bad",
+                  "OK"
+                ]
+              },
+              {
+                "label": "Other",
+                "values": [
+                  "Other"
+                ]
+              }
+            ],
+            "defaultCategory": {
+              "label": "Unknown"
+            }
+          },
+          "Status_Long": {
+            "kind": "categorical",
+            "value": {
+              "tsx": "tolong($event.[Status].Double)"
```

Comment:
update TSX, maybe show this example with $event['Status'].Long cause we have long support now :)